bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTableModel.scala

```diff
@@ -150,41 +151,58 @@ sealed trait ItcTableModel extends AbstractTableModel {
+    Column("S/N Total",       (c, i, r) => imgTotalSNRatio (r))
   )
 }
 
 /** GMOS specific ITC imaging table model. */
-class ItcGmosImagingTableModel(val keys: List[ItemKey], val uniqueSteps: List[ItcUniqueConfig], val res: List[Future[ItcService.Result]]) extends ItcImagingTableModel {
+class ItcGmosImagingTableModel(val keys: List[ItemKey], val uniqueSteps: List[ItcUniqueConfig], val inputs: List[String\/ItcInputs], val res: List[Future[ItcService.Result]]) extends ItcImagingTableModel {
   val headers = Headers
   val results = List(
-    Column("CCD1 Peak",       (c, r) => imgPeakPixelFlux(r, ccd=0),   tooltip = ItcTableModel.PeakPixelTooltip + " for CCD 1"),
-    Column("CCD1 S/N Single", (c, r) => imgSingleSNRatio(r, ccd=0)),
-    Column("CCD1 S/N Total",  (c, r) => imgTotalSNRatio (r, ccd=0)),
-    Column("CCD2 Peak",       (c, r) => imgPeakPixelFlux(r, ccd=1),   tooltip = ItcTableModel.PeakPixelTooltip + " for CCD 2"),
-    Column("CCD2 S/N Single", (c, r) => imgSingleSNRatio(r, ccd=1)),
-    Column("CCD2 S/N Total",  (c, r) => imgTotalSNRatio (r, ccd=1)),
-    Column("CCD3 Peak",       (c, r) => imgPeakPixelFlux(r, ccd=2),   tooltip = ItcTableModel.PeakPixelTooltip + " for CCD 3"),
-    Column("CCD3 S/N Single", (c, r) => imgSingleSNRatio(r, ccd=2)),
-    Column("CCD3 S/N Total",  (c, r) => imgTotalSNRatio (r, ccd=2))
+    Column("CCD1 Peak",       (c, i, r) => imgPeakPixelFlux(r, ccd=0),   tooltip = ItcTableModel.PeakPixelTooltip + " for CCD 1"),
+    Column("CCD1 S/N Single", (c, i, r) => imgSingleSNRatio(r, ccd=0)),
+    Column("CCD1 S/N Total",  (c, i, r) => imgTotalSNRatio (r, ccd=0)),
+    Column("CCD2 Peak",       (c, i, r) => imgPeakPixelFlux(r, ccd=1),   tooltip = ItcTableModel.PeakPixelTooltip + " for CCD 2"),
+    Column("CCD2 S/N Single", (c, i, r) => imgSingleSNRatio(r, ccd=1)),
+    Column("CCD2 S/N Total",  (c, i, r) => imgTotalSNRatio (r, ccd=1)),
+    Column("CCD3 Peak",       (c, i, r) => imgPeakPixelFlux(r, ccd=2),   tooltip = ItcTableModel.PeakPixelTooltip + " for CCD 3"),
+    Column("CCD3 S/N Single", (c, i, r) => imgSingleSNRatio(r, ccd=2)),
+    Column("CCD3 S/N Total",  (c, i, r) => imgTotalSNRatio (r, ccd=2))
   )
 }
 
+class ItcGsaoiImagingTableModel(val keys: List[ItemKey], val uniqueSteps: List[ItcUniqueConfig], val inputs: List[String\/ItcInputs], val res: List[Future[ItcService.Result]]) extends ItcImagingTableModel {
+  val headers = HeadersWithCoadds ++ List(
+    Column("Strehl",          (c, i, r) => gems(i),                      tooltip = "Estimated Strehl and band")
+  )
+  val results = List(
+    Column("Peak",            (c, i, r) => imgPeakPixelFlux(r),          tooltip = ItcTableModel.PeakPixelTooltip),
+    Column("S/N Single",      (c, i, r) => imgSingleSNRatio(r)),
+    Column("S/N Total",       (c, i, r) => imgTotalSNRatio (r))
+  )
+
+  def gems(i: String \/ ItcInputs): Option[String] = i.toOption.map { inputs =>
+    val gems = inputs.ins.asInstanceOf[GsaoiParameters].gems
```

Comment:
I got confused about this `inputs` val and the `ins` method with  the class param `inputs` perhaps using different names would make the code clearer